### ISSUE TITLE
Not an issue, just questions about blink16

### ISSUE THREAD

**Issue by Vutshi** [ghaerr/blink16#5]
Created to discuss things I don't understand about blink16.

1. I found that the following code is shown as bipartite in blink16:
```
1000:0126 26                es                
1000:0127 8b 07             mov     (%bx),%ax 
```
whereas MartyPC and Ghidra show it as one instruction:
<img width="379" alt="Screenshot 2024-05-30 at 22 31 44" src="https://github.com/ghaerr/blink16/assets/4971779/33a870ed-03fd-43d2-af15-a2b1e6ad4c82">

Is this intended behavior?

2. Is there a way to switch the memory views to the actual byte values?

Best

**Comment by ghaerr** [comment]
> I found that the following code is shown as bipartite in blink16:
> Is this intended behavior?

Yes - I grabbed the disassembler I had written for [86sim](https://github.com/ghaerr/86sim) and it displays instruction prefixes on a separate line. Its on my list of things to enhance, but initially proved to be a bit more complicated than I had hoped. Both the 8086 emulator and disassembler are extremely small, only a single .c file each.

BTW, has the AT&T vs Intel 8086 instruction display difference driven you crazy yet? I suppose it would also be nice to have an Intel instruction display option.

> Is there a way to switch the memory views to the actual byte values?

That is also on my list, and would be a nice enhancement.

Let me see what I can do...

**Comment by Vutshi** [comment]
> BTW, has the AT&T vs Intel 8086 instruction display difference driven you crazy yet?

Yes, it gets on my nerves. I find Intel representation much cleaner because `%%%` hurt my eyes. On the other hand, the backward thinking in Intel seems strange to me.

> I suppose it would also be nice to have an Intel instruction display option.

That would be cool.
